src/middleware.ts

```diff
@@ -0,0 +1,41 @@
+import { Client } from './client';
+
+export function expiredToken(client: Client, retries = 5) {
+  return client.instance.interceptors.response.use(undefined, async error => {
```

Comment:
Trusting that axios knows to ignore `undefined` here (just wasn't in their limited set of examples, but then neither was the error handling function 🔥

Reply:
@rrsmith i trust the types :)